package.json

```diff
@@ -8,9 +8,12 @@
   "repository": "https://github.com/thysultan/stylis.js",
   "bugs": "https://github.com/thysultan/stylis.js/issues",
   "type": "module",
-  "main": "dist/stylis.umd.js",
+  "main": "dist/umd/stylis.umd.js",
   "module": "dist/stylis.esm.js",
-  "exports": "./index.js",
+  "exports": {
+    "import": "./index.js",
+    "require": "./dist/umd/stylis.umd.js"
```

Comment:
Why can't this point directly to the file?

Reply:
By directly i mean directly to the current file `/dist/stylis.umd.js`

Reply:
For example in https://nodejs.org/api/esm.html#esm_approach_2_isolate_state they don't try to isolate the cjs file into a directory.

Reply:
> Why can't this point directly to the file? Doesn't the require imply cjs.

Unfortunately not, this is also kinda quirky for me and counter-intuitive. `"require"`/`"import"` keys only tell which loader should be used, but import is able to load cjs files, so it can't set the mode of the interpretation of the imported file. Maybe it's just a symmetry thing for `"require"`?

> For example in https://nodejs.org/api/esm.html#esm_approach_2_isolate_state they don't try to isolate the cjs file into a directory.

Yes, but they use `.cjs` extension - which sets the mode of the interpretation. `.mjs` files are always modules, `.cjs` are always scripts. I've just wanted to avoid `.cjs` extension here, especially given the fact that this is a UMD bundle which often is used when trying to use a particular lib directly in a browser, and `.cjs` doesn't make much sense in such context. We can, however, use `.cjs` to avoid this extra directory.

To sum it up:
- "require"/"import" - don't do anything in regards to module/script interpretation
- `.mjs`/`.cjs` extensions force a particular mode
- `.js` are parsed based on the nearest package.json and its "type", that's why we can create a "nested package scope", as the algorithm allows this, even though this section doesn't really mention this explicitly: https://nodejs.org/api/esm.html#esm_package_scope_and_file_extensions . This technique is used by some packages so a `.js` extension can be used for all files (for example Rollup does this - https://github.com/rollup/rollup/blob/c0c206e2a8a892fdb0e4f9b25b0a817b733a072f/rollup.config.js#L118 )

Reply:
>We can, however, use .cjs to avoid this extra directory.

If we can avoid this, let us export a .cjs file, i think rollup can produce a dedicated cjs file without the wrappers involved in the umd file.

Reply:
Ok, added a separate config item for `.cjs` file